components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2965,11 +2905,11 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         try {
             await this.teamDB.transaction(async (db) => {
                 await db.removeMemberFromTeam(userToBeRemoved.id, orgID);
-                await this.authorizer.writeRelationships(removeUserFromOrg(orgID, userId));
+                await this.authRelationships.removeUserFromOrg(orgID, userId);
             });
         } catch (err) {
             // Rollback to the original role the user had
-            await this.authorizer.writeRelationships(organizationRole(orgID, userId, membership.role));
+            await this.authRelationships.organizationRole(orgID, userId, membership.role);
```

Comment:
Asymmetry between `remove` and `add`. We should have consistent action keywords between these

Reply:
You mean name the writes like `addOrganizationRole`?

Reply:
Yep, something like that